### PR TITLE
Install latest stable go for travis and reformat bzl files

### PR DESCRIPTION
**Pull request by ixdy**
Maybe this will work? Ultimately we should convert this to a prow job, but this is initially the least amount of effort needed to get things building again, I think.

/assign @BenTheElder

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,12 @@
 dist: trusty
 sudo: required
 
-# Install Bazel and set up GOPATH.
+# Install latest Go and Bazel and set up GOPATH.
 before_install:
+# gimme on travis is too old for .x version resolution
```
Comment (BenTheElder) [inline .travis.yml]: :this_is_fine:

**Comment by ixdy** [comment]
/unassign @BenTheElder 
I'm still experimenting here.

**Comment by ixdy** [comment]
The latest buildifier has *opinions* about how bzl files should be formatted, so I've fixed those. (It's been getting annoying that I'm getting everything reformatted when I update files in vim, anyhow.)
(We should probably pin a particular version of buildifier...)

/assign @BenTheElder

**Comment by ixdy** [comment]
oh good, they fixed that multiline comment bug.

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,11 @@
 dist: trusty
 sudo: required
 
-# Install Bazel and set up GOPATH.
+# Install latest Go and Bazel and set up GOPATH.
 before_install:
+# gimme on travis is too old for .x version resolution, and gimme stable seems broken, too
+- LATEST_GO=$(gimme --known | sort -V | tail -1)
+- eval "$(gimme ${LATEST_GO})"
```
[the remark this answers is not in the document]
Comment (ixdy) [reply .travis.yml]: it's the recommended practice, per https://github.com/travis-ci/gimme#installation--usage

[the remark this answers is not in the document]
Comment (BenTheElder) [reply .travis.yml]: these two lines are just "Very bash" :^)

[the remark this answers is not in the document]
Comment (BenTheElder) [reply .travis.yml]: I'm not actually worried about them, just sarcastically 🙃

**Pull request merged**